### PR TITLE
http obfuscation update

### PR DESCRIPTION
**Pull request by ajgajg1134**
Customer recently had some confusion here, this change makes it clearer that http obfuscation applies to both `http` and `web` type spans. Additionally clarifies that remove_paths_with_digits ONLY works on digits, any path segment with both digits and letters will not be redacted.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/configure_data_security/_index.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ apm_config:
 ```
 
 * `remove_query_string`: If true, obfuscates query strings in URLs.
-* `remove_paths_with_digits`: If true, path segments in URLs containing digits are replaced by "?".
+* `remove_paths_with_digits`: If true, path segments in URLs containing only digits are replaced by "?".
```
Comment (wantsui) [inline content/en/tracing/configure_data_security/_index.md:265]: Does it look for the span tag `http.url` ?

**Review by wantsui — APPROVED** [review]
Looks good to me, I just left a small suggestion to clarify which span tag is used for URLs!

File: content/en/tracing/configure_data_security/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cswatt) [reply content/en/tracing/configure_data_security/_index.md]: applied this change!

**Review by cswatt — APPROVED** [review]

File: content/en/tracing/configure_data_security/_index.md [hunk 2]
```diff
@@ -262,7 +262,7 @@ apm_config:
 ```
 
 * `remove_query_string`: If true, obfuscates query strings in URLs.
```
Comment (wantsui) [inline content/en/tracing/configure_data_security/_index.md:264]: Should this one also give a note that it's looking at `http.url`?

Comment (ajgajg1134) [inline content/en/tracing/configure_data_security/_index.md:264]:
```suggestion
* `remove_query_string`: If true, obfuscates query strings in URLs (`http.url`).
```

File: content/en/tracing/configure_data_security/_index.md [hunk 3]
```diff
@@ -261,8 +261,8 @@ apm_config:
       remove_paths_with_digits: true
 ```
 
-* `remove_query_string`: If true, obfuscates query strings in URLs.
-* `remove_paths_with_digits`: If true, path segments in URLs containing digits are replaced by "?".
+* `remove_query_string`: If true, obfuscates query strings in URLs (`http.url`).
+* `remove_paths_with_digits`: If true, path segments in URLs (`http.url`) containing only digits are replaced by "?".
```
Comment (ajgajg1134) [inline content/en/tracing/configure_data_security/_index.md:265]:
```suggestion
* `remove_query_string` or environment variable `DD_APM_OBFUSCATION_HTTP_REMOVE_QUERY_STRING`: If true, obfuscates query strings in URLs (`http.url`).
* `remove_paths_with_digits` or environment variable `DD_APM_OBFUSCATION_HTTP_REMOVE_PATHS_WITH_DIGITS`: If true, path segments in URLs (`http.url`) containing only digits are replaced by "?".
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**